global.json

```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "8.0.100-alpha.1.22531.1"
+    "version": "8.0.100-alpha.1.23054.7"
   },
   "tools": {
-    "dotnet": "8.0.100-alpha.1.22531.1",
```

Comment:
Because of the new SDK, I believe we will need to update the CI pipeline to use the `binlogtosln` that supports `v15` (https://github.com/dotnet/arcade/issues/11995), if it is not updated yet.

Reply:
Just merged that PR into this one